lib/vrl/stdlib/src/match_any.rs

```diff
@@ -0,0 +1,116 @@
+use regex::bytes::RegexSet;
+use vrl::prelude::*;
+
+#[derive(Clone, Copy, Debug)]
+pub struct MatchAny;
+
+impl Function for MatchAny {
+    fn identifier(&self) -> &'static str {
+        "match_any"
+    }
+
+    fn parameters(&self) -> &'static [Parameter] {
+        &[
+            Parameter {
+                keyword: "value",
+                kind: kind::BYTES,
+                required: true,
+            },
+            Parameter {
+                keyword: "patterns",
+                kind: kind::ARRAY,
+                required: false,
```

Comment:
Why would this not be required?

Reply:
Good catch! It should definitely be required. Seems like this was somehow fixed when rebasing.

Reply:
Doh, I was looking at somewhere else. Will push a fix now.